### ISSUE TITLE
compile

### ISSUE THREAD

**Issue by kylliancop** [hackerized1/DeadSide-GameCheat-External-AlienCheats#1]
how to compile?

**Comment by kylliancop** [comment]
unable to compile

**Comment by kylliancop** [comment]
Gravité	Code	Description	Projet	Fichier	Ligne	État de la suppression
Erreur (active)	E1696	impossible d'ouvrir le fichier source "d3dx9.h"	DeadSide	C:\Users\kylli\Desktop\DeadSide-main\DeadSide\overlay.h	4	
Erreur (active)	E0020	identificateur "D3DXMATRIX" non défini	DeadSide	C:\Users\kylli\Desktop\DeadSide-main\DeadSide\struct.h	384	
Erreur	C1083	Impossible d'ouvrir le fichier include : 'd3dx9.h' : No such file or directory	DeadSide	C:\Users\kylli\Desktop\DeadSide-main\DeadSide\overlay.h	4

**Comment by hackerized1** [comment]
VS 2019 and wdk install https://github.com/hackerized1/VisualStudio2019-wdksetup2

**Comment by kylliancop** [comment]
yes for that I have installed wdk but there are still plenty of other problems

**Comment by kylliancop** [comment]
you had discord ?

**Comment by hackerized1** [comment]
yes         
!                     Hackerized#2476

**Comment by kylliancop** [comment]
your discord is not working here is mine BX#0941

**Comment by kylliancop** [comment]
a comment on the opening?

**Comment by kylliancop** [comment]
how about the opener?

**Comment by hackerized1** [comment]
Hackerized#2476

**Comment by kylliancop** [comment]
don't work

**Comment by kylliancop** [comment]
add me on discord BX#0941

**Comment by kylliancop** [comment]
????
